roles/insert_kernel_module/tasks/insert_module.yml

```diff
@@ -0,0 +1,32 @@
+# Detect what type of kernel module it is - builtin, loadable, or missing
+- name: "Import tasks to detect module type : {{ insert_kernel_module_data.name }}"
+  ansible.builtin.import_tasks: detect_module.yml
+
+# Sometimes it might be preffered to use the off-tree module instead of the module
+# that is already shipped with the kernel. When a force_build flag is set for
+# a module a build_module.yml will be called instead of loading it.
+# NOTE(r-krcek): This needs testing, I am not sure if it works as-is!!
```

Comment:
to be tested and this line to be removed then

Reply:
Tested by Roman, needs testing by third party